SUMMARY.md

```diff
@@ -27,6 +26,14 @@
     * [Khala Governance](overview/pha-token/governance/khala-governance.md)
     * [Setting Up an Account Identity](overview/pha-token/governance/setup-account-identity.md)
 
+## Confidential AI Reference
```

Comment:
```suggestion
## Confidential AI
```

Reply:
Should be "Inference"

Reply:
Explicitly saying Inference will make customers more clear that we only do AI reference in TEE, bc we do not support training.
"Reference" is a typo here, my bad